### PR TITLE
(BOLT-578) Report bundled content to GA

### PR DESCRIPTION
**Pull request by donoghuc**
Include the name of the task/plan being run if that name refers to a piece of built-in content. The task/plan name should not be included if it isn't a piece of bundled content, as the name may be sensitive.

### CODE REVIEW

**Comment by donoghuc** [comment]
Plan to move logic such that tasks/plans not directly called via CLI will be captured (possibly in run_task?).

File: lib/bolt/executor.rb [hunk 1]
```diff
@@ -127,6 +146,13 @@ def report_function_call(function)
       @analytics&.event('Plan', 'call_function', function)
     end
 
+    def report_bundled_content(mode, name)
+      if BUNDLED_CONTENT.include?(name)
+        screen = "bundled_#{mode}_content"
+        @analytics&.screen_view(screen, bundled_content: name)
```
Comment (MikaelSmith) [inline lib/bolt/executor.rb]: This makes more sense as an event, similar to calling a function or using a transport. Could be something like `event('Bundled Content', mode, name)`.

**Comment by MikaelSmith** [comment]
It looks like somehow when you updated you pulled in all the changes from https://github.com/puppetlabs/bolt/commit/f16c05f68d6cc5bcb81eedd0db2fb705fb395c0c.

**Comment by MikaelSmith** [comment]
A `git reset --hard upstream/master && git cherry-pick 44ace3194511679343bf22c7a523d75f04b58e50` would probably fix that.

**Comment by donoghuc** [comment]
Yes that was a mistake. I will clean that up.

File: lib/bolt/cli.rb [hunk 2]
```diff
@@ -353,5 +357,16 @@ def file_stat(path)
     def outputter
       @outputter ||= Bolt::Outputter.for_format(config[:format], config[:color], config[:trace])
     end
+
+    def load_bundled_content
+      default_content = Bolt::PAL.new(Bolt::Config.new)
```
Comment (MikaelSmith) [inline lib/bolt/cli.rb]: It looks like this should avoid picking up the default modulepath and just use the vendored modules. It also doesn't appear to have any adverse impact on performance. So this approach seems ok.

I'd rename this to `bundled_content` and just have it called directly when constructing the Executor. No need to have separate checks beforehand.

Comment (donoghuc) [reply lib/bolt/cli.rb]: Oh right, task/plan are mutually exclusive on CLI. Will clean that up.

**Pull request merged**